sql/migrations/state/0020_atx_weight.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE atxs ADD COLUMN weight INTEGER;
+INSERT INTO atxs (weight) SELECT effective_num_units * tick_count FROM atxs;
```

Comment:
This should be:
```suggestion
UPDATE atxs SET weight = effective_num_units * tick_count;
```

Note: this migration takes around 50s on a recent database on an NVME drive.